### ISSUE TITLE
Redesign Tab Wrangler's brand

### ISSUE THREAD

**Issue by ssorallen** [tabwrangler/tabwrangler#168]
Any help on this issue from designers or other developers would be greatly appreciated! I'm happy to add your name to the contributors list and in the Chrome Webstore description.

Tab Wrangler's Webstore presence uses lo-res screenshots from previous versions, and TW's icon feels outdated. A refresh would make TW appear more professional and maintained.

Necessary:

- [x] Update the screenshots in the Chrome Webstore: https://chrome.google.com/webstore/detail/tab-wrangler/egnjhciaieeiiohknchakcodbpgjnchh
- [ ] Update the description in the Chrome Webstore to more accurately describe the extension
- [ ] Create a new icon (including a hi-res one) for use both in the Webstore and in Chrome itself (#180)
- [ ] Create [promotional images](https://developer.chrome.com/webstore/images#promo) for use in the Chrome Webstore (small, large, and marquee)

**Comment by Voknehzyr** [comment]
I tried to make new icon design in Flat style.
If you wanna use it I can give you a SVG or AI file.
![tabwranglericonbeta](https://user-images.githubusercontent.com/10144091/36164715-435fc3be-10fe-11e8-97e2-56d414d39c74.png)

**Comment by ssorallen** [comment]
Here's an idea I'm messing around with:

![tab-wrangler](https://user-images.githubusercontent.com/29612/37518246-594d12a8-294f-11e8-9b68-f266a382b2b9.png)

**Comment by ssorallen** [comment]
@Voknehzyr I like yours! Some of the detail would be lost when it is shrunk to the size used by Chrome though. It would also be great if it fill more of a square in order to use as much space as possible. The [app icon](https://developer.chrome.com/webstore/images#icons) docs has good advice on icons.

**Comment by ssorallen** [comment]
This one explains a bit more what the extension does:

![tab-wrangler-v2](https://user-images.githubusercontent.com/29612/37551554-b8e1ccd0-29dc-11e8-8a8f-333019fcadfb.png)

**Comment by Voknehzyr** [comment]
@ssorallen I think that your icon will be too big, actually, I'm thinking about making icon in pixel art style like that we have now.

**Comment by ssorallen** [comment]
@Voknehzyr Yes, that image I uploaded would be way too big for an icon! That image was one of the [promotional image sizes](https://developer.chrome.com/webstore/images#promo).

> Small: 440x280 pixels (required)
> Large: 920x680 pixels
> Marquee: 1400x560 pixels

**Comment by ssorallen** [comment]
I was considering keeping the current app icon but updating it a bit. The 440x280 image I uploaded was my way of integrating the app icon into a promotional image.

**Comment by ggabogarcia** [comment]
Hello, here is my logo proposal, I hope you like it, any suggestion let me know. 

![tab waleder](https://user-images.githubusercontent.com/35750428/44064868-13867846-9f2d-11e8-9cc8-01f2aeb4165c.jpg)

**Comment by ssorallen** [comment]
@ggabogarcia This looks great, thank you! I wonder if the shadow could be smaller on the background blue? A flatter look seems to fit with other logos like [HTTPS Everywhere](https://www.eff.org/https-everywhere), [uBlock Origin](https://github.com/gorhill/uBlock), [Grammarly](https://chrome.google.com/webstore/detail/grammarly-for-chrome/kbfnbcaeplbcioakkpcpgfkobkghlhen?authuser=0&hl=en).

**Comment by ggabogarcia** [comment]
Hello, here is the same concept with the flat shadow. :)

![tab waleder2](https://user-images.githubusercontent.com/35750428/44112338-76320102-9fca-11e8-9eee-f5541e826ee9.jpg)

**Comment by ggabogarcia** [comment]
Other :)

![tab waleder3](https://user-images.githubusercontent.com/35750428/44120573-650cdd4e-9fe2-11e8-849c-f403a2a94f0a.jpg)

**Comment by ssorallen** [comment]
@ggabogarcia These look fantastic! Thank you.

I would choose the gray X over the green one because the green one looks too busy when the icon is small. The gray X is still quite clear at a small size.

I quite like the black outline one too. What do you think about making that black outline a little thinner?

Which ones do you like the most?

**Comment by Voknehzyr** [comment]
I like first logo more than others, the only one thing I would change is make shadow shorter.
Variant with green button looks pretty strange tbh.

**Comment by ggabogarcia** [comment]
Here with the thinnest outline in black, and without it, I really like the black outline, it has more visual strength, but the final decision is yours :).

![tab waleder3](https://user-images.githubusercontent.com/35750428/44177132-29c1f500-a0b2-11e8-9347-1129fce048c1.jpg)

**Comment by ggabogarcia** [comment]
I would also like to know what they have in mind. In addition to representing through a logo how the extension works, it is quite complicated.

**Comment by ingorichter** [comment]
The designs are amazing. My favorite is the one without the black outline. The one with the black outline seems a bit too thick for my taste and I’m not sure how it’ll look when the icon is in the toolbar next to all the other extension icons.

**Comment by ggabogarcia** [comment]
Proof of size :)

![tab waleder4](https://user-images.githubusercontent.com/35750428/44179753-60057180-a0be-11e8-8100-2f7247781a40.jpg)

**Comment by ssorallen** [comment]
@ggabogarcia Since your original design did not include the black outline and both both @Voknehzyr and @ingorichter like the design with no outline, let’s go with it! The version in your last post looks really great. I will add the icons to the extension.

Do you prefer a specific way of being attributed? A link to your website or to your GitHub profile?

If you have interest in doing larger designs, the Chrome Web store also asks for promotional images in a few sizes: https://developer.chrome.com/webstore/images#promo

Of course there’s no need to do anything else. I am already grateful for the fresh icon!

**Comment by ggabogarcia** [comment]
Excellent, I will prepare the files to send them :), svg, png and more.

**Comment by ggabogarcia** [comment]
Hello, here are all the files, I hope all this well :)
### [FILES](https://drive.google.com/drive/folders/1expeXmnK3wReS-J3k7MOImue-YFcfLCt?usp=sharing)
![DERECHOS.jpg](https://cdn.steemitimages.com/DQmVSH7hxt3nuyDxNRdWUBUBtfR11TyvLxM1F1C4vYW5UzG/DERECHOS.jpg)
This work is licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/)

**Comment by ssorallen** [comment]
@ggabogarcia Thank you for the files! The Chrome Extension page about icons mentions the following:

> The actual icon size should be 96x96 (for square icons); an additional 16 pixels per side should be transparent padding, adding up to 128x128 total image size.

Would you be able to adjust based on those notes? I also think to support retina 2x sizes, I will use double the size for each one they require, so 256x256 for the 128x128 size, etc.

**Comment by ggabogarcia** [comment]
I don't understand, I made icons for other extensions and they did not tell me about that margin of 16 pixels, always guide me through [this](https://developer.chrome.com/apps/manifest/icons). It would be better to tell me what are the sizes you need. :smile:

**Comment by ssorallen** [comment]
@ggabogarcia Oh okay, no worries. I'll use the assets you created. I trust you, I haven't updated an icon before.

**Comment by ssorallen** [comment]
@ggabogarcia The PNGs are named "256x256", "128x128", etc. but they are actually only the height of the icon. The "256x256", for example, is actually 256x160. The SVGs appear to have padding above and below, but extensions require PNGS.

Should I pad those images with vertical space myself?

**Comment by ggabogarcia** [comment]
@ssorallen I got it, I will make it darker and more square. The name of the files put them that way, but certainly, the form was horizontal, so those spaces were always going to be :). I'll fix it anyway

**Comment by ggabogarcia** [comment]
@ssorallen The files are updated, I made the icon more square, but without abusing its shape, the gray color is darker and the "X" also, I hope it works this time :+1:

**Comment by ggabogarcia** [comment]
The icon should not deform, you can see the difference below

![44813220-ed58c380-ab9e-11e8-8002-34435179d75d](https://user-images.githubusercontent.com/35750428/44823004-800b5980-abc3-11e8-962f-e2701ea78706.png)


![44813220-ed58c380-ab9e-11e8-8002-34435179d75d1](https://user-images.githubusercontent.com/35750428/44823012-8699d100-abc3-11e8-8380-a5806eb08713.png)


Here the test, the file that I sent is the second image, the first one is distorted. Can you see it? It is not recommended to stretch, I know you want to cover all the space, but it is not an obligation. But if you want the first proposal, I will do it. :+1:

**Comment by ssorallen** [comment]
I agree, it should not be distorted! It doesn’t need to cover all the space. That was a bad suggestion, let’s go with your original design.

**Comment by ggabogarcia** [comment]
Hello, the files are ready :)

**Comment by ssorallen** [comment]
What do you all think with the icons in place?:

URL bar:
<img width="222" alt="screen shot 2018-09-05 at 11 59 18 am" src="https://user-images.githubusercontent.com/29612/45105876-4bd3f380-b103-11e8-950b-6556a9ee0a9b.png">

Extensions page:
<img width="866" alt="screen shot 2018-09-05 at 11 59 49 am" src="https://user-images.githubusercontent.com/29612/45105877-4c6c8a00-b103-11e8-898a-a7844221b185.png">

**Comment by IfeanyiEkperi** [comment]
Wow great. I didn't see this have been resolved, I went on to design a logo.

Nice Designs....

**Comment by ssorallen** [comment]
As I've used Chrome + Firefox since we first talked about this ages ago I've decided that (1) extension icons should fill a square as much as possible and (2) icons don't need a skeumorphic cues as to what the extension does. The icon needs to convey the unique brand.

@ggabogarcia I appreciate your work from back in 2018 but am going to look at a brand + icon that don't necessarily try to represent a browser tab in a 32x32 square.

Since this issue has been open for ages I am going to close it and track progress of a new icon in another issue.
